keyboards/teahouse/ayleen/ayleen.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 Freather
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+#define LAYOUT( \
```

Comment:
this can be moved into the info.json.

You can preview/generate that by running: 
```sh
qmk info -kb teahouse/ayleen -f json
```

Reply:
the file has removed and merged into info.json